frontend/components/App/App.tsx

```diff
@@ -16,7 +18,10 @@ import TableProvider from "context/table";
 import QueryProvider from "context/query";
 import PolicyProvider from "context/policy";
 import { AppContext } from "context/app";
-import FleetErrorBoundary from "pages/errors/FleetErrorBoundary";
+
+import { ErrorBoundary } from "react-error-boundary"; // @ts-ignore
+import Fleet404 from "pages/errors/Fleet404"; // @ts-ignore
+import Fleet500 from "pages/errors/Fleet500";
```

Comment:
Not relevant to this PR, but it would be great to add a 403 error overlay at some point.

Reply:
Oh yeah that'll be done for #3403.